packages/suite/src/hooks/wallet/useCoinmarketSpend.ts

```diff
@@ -42,11 +43,15 @@ const useSpendState = ({ selectedAccount, fees }: Props, currentState: boolean)
 export const useCoinmarketSpend = (props: Props): SpendContextValues => {
     const [voucherSiteUrl, setVoucherSiteUrl] = useState<string | undefined>('error');
 
-    const { addNotification, setShowLeaveModal, saveTrade } = useActions({
+    const { addNotification, setShowLeaveModal, saveTrade, loadInvityData } = useActions({
         addNotification: notificationActions.addToast,
         setShowLeaveModal: coinmarketSellActions.setShowLeaveModal,
         saveTrade: coinmarketSpendActions.saveTrade,
+        loadInvityData: coinmarketCommonActions.loadInvityData,
     });
+
+    loadInvityData();
```

Comment:
what about to call it only once not on every re-render?
```suggestion
    useEffect(() => {
      loadInvityData();
    }, [loadInvityData]);
```

Reply:
Fixed.